### PR TITLE
docs - note OOM query failure when resource groups active

### PR DESCRIPTION
**Pull request by lisakowen**
add a note that transaction will fail if no memory available in resource group.

doc review site link:  http://docs-gpdb-review-staging.cfapps.io/550/admin_guide/workload_mgmt_resgroups.html#topic8339717 (note at end of section)

### CODE REVIEW

**Comment by mkiyama** [comment]
I think this should be mentioned as a difference between how resource groups and resource queues terminates queries.

I believe a resource group terminates the query that requests the memory that exceeds available memory.

This is a description of how resource queues terminates queries from the GUC description.

http://gpdb.docs.pivotal.io/550/ref_guide/config_params/guc-list.html#runaway_detector_activation_percent

> Greenplum Database terminates queries based on memory usage, starting with the query consuming the largest amount of memory. Queries are terminated until the percentage of utilized vmem is below the specified percentage.

**Comment by lisakowen** [comment]
thanks for reviewing, @gaos1 .  does the new text more accurately describe the situation?

File: gpdb-doc/dita/admin_guide/wlmgmt.xml [hunk 1]
```diff
@@ -80,7 +80,7 @@
           <row>
             <entry colname="col1">Query Failure</entry>
             <entry colname="col2">Query may fail immediately if not enough memory</entry>
-            <entry colname="col3">Query may fail after reaching transaction memory limit</entry>
+            <entry colname="col3">Query may fail after reaching transaction fixed memory limit, no shared resource group memory exits, and the transaction requests more memory</entry>
```
Comment (dyozie) [inline gpdb-doc/dita/admin_guide/wlmgmt.xml]: Having trouble parsing this.  Might be a typo also with "exits."  Maybe this should say:

"Query may fail after reaching the transaction fixed memory limit if no shared resource group memory exists and the transaction requests more memory."

**Comment by gaos1** [comment]
Looks good. Thanks.

2018年2月22日 上午10:06，"Lisa Owen" <notifications@github.com>写道：

thanks for reviewing, @gaos1 <https://github.com/gaos1> . does the new text
more accurately describe the situation?

—
You are receiving this because you were mentioned.

Reply to this email directly, view it on GitHub
<https://github.com/greenplum-db/gpdb/pull/4568#issuecomment-367532696>,
or mute
the thread

**Pull request merged**